aws_setup.sh

```diff
@@ -34,7 +34,8 @@ CLOUDSWEEPER_POLICY='{
                 "s3:GetBucketLocation",
                 "s3:PutBucketTagging",
                 "s3:DeleteObject",
-                "s3:DeleteBucket"
+                "s3:DeleteBucket",
```

Comment:
we gave you two permissions during prototyping, do you not need listMetrics anymore for this version? 

"cloudwatch:GetMetricStatistics", "cloudwatch:ListMetrics"

Reply:
I'm not 100% sure, but I think `ListMetrics` was only needed for me to able to browse the AWS Console to figure what Metrics I actually needed. now that I have determined that, only `GetMetricStatistics` is needed. but if I'm wrong, it should be easy enough to add back :)

from what I could tell, the API calls I do only needs the `GetMetricStatistics`